assets/js/templates/snippet/image.html

```diff
@@ -0,0 +1,6 @@
+<!-- File Button -->
+<div class="control-group">
+  <label class="control-label"
+         for="<%= field_guid %>"><%= label %></label>
+  <button id="<%= field_guid %>" name="<%= field_guid %>">Upload</button>
```

Comment:
Hmm, I don't think we translate any of the text in any of the other snippets. Maybe that's something we should look into doing.